### PR TITLE
[Xedra Evolved] Typos bad, give vampire CBMs non-placeholder descriptions

### PR DESCRIPTION
**Pull request by TheShadowFerret**
#### Summary
None

#### Purpose of change

I sees typos, two CBMs use copied vanilla CBM descriptions.

#### Describe the solution

Improve da gramma, give Batsignal CBMs a real description

#### Describe alternatives you've considered

Not change structure of bat roost's description.

#### Testing

N/A

#### Additional context

~~*Pokes a certain person to learn his its/it's*~~

### CODE REVIEW

**Review by Maleclypse — APPROVED** [review]
Just saw this after posting mine. And saw you adjust the new CBM descriptions in real time.  Though I think I might have liked your original CBM names better.

edit: Yes ignore my new names for them because I'm just rushing to get a fix out.  I'll remove my changes to the renfield cbm file so we aren't conflicting.

**Comment by TheShadowFerret** [comment]
> Though I think I might have liked your original CBM names better.

They were yours, I copied them from the item form 🤔

**Comment by Maleclypse** [comment]
> > Though I think I might have liked your original CBM names better.
> 
> They were yours, I copied them from the item form 🤔

Even better! My genius shines through my mistakes. /s Lol thank you so much for your help on fixing these.

File: data/mods/Xedra_Evolved/player/renfield_cbms.json [hunk 1]
```diff
@@ -59,8 +59,8 @@
   {
     "id": "xe_bio_xl_chiropteran_signal",
     "type": "bionic",
-    "name": { "str": "Fingerpick" },
-    "description": "One of your fingers has an electronic lockpick surgically embedded in it.  This automatic system will quickly unlock all but the most advanced key locks without any skill required on the part of the user.",
+    "name": { "str": "Desmodontinae Gargantua Vocalizer" },
+    "description": "Your vocal cords have been reinforced and enlarged. You can make a strange noise with it, but you don't know what it's supposed to communicate.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/player/renfield_cbms.json:63]:
```suggestion
    "description": "Your vocal cords have been reinforced and enlarged.  You can make a strange noise with it, but you don't know what it's supposed to communicate.",
```

**Pull request merged**